dev/com.ibm.ws.jaxrs.2.1_fat_executor/test-applications/jaxrsapp/src/web/jaxrstest/JAXRSExecutorTestServlet.java

```diff
@@ -62,6 +62,16 @@ public class JAXRSExecutorTestServlet extends FATServlet {
 
     @Resource(name = "java:comp/env/concurrent/scheduledExecutorRef", lookup = "concurrent/scheduledExecutor")
     private ManagedScheduledExecutorService scheduledExecutor;
+    
+    private static boolean ee10;
+    static {
+       try {
+           Class.forName("org.jboss.resteasy.plugins.providers.multipart.ResteasyEntityPartBuilder");
```

Comment:
Why this class instead of a jakarta.ws.rs class that is new in EE10.  I think you are slightly lucky that this class is found because it isn't actually an API that is supposed to be loadable by applications normally.